extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/DontRollback.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.narayana.jta;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * If you annotate your exception with this annotation, the transactional interceptor will not
+ * rollback when your exception it thrown.
+ * 
+ * FIXME: move to SPI so it can be used by TransactionalReactive
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface DontRollback {
```

Comment:
I think this should also support forcing a rollback (@Rollback(true) and @Rollback(false)). 

At the moment rollback happens for unchecked exceptions, and it does not roll back for checked, so we should be able to override both these behaviors.

Reply:
OK done.